### PR TITLE
First page

### PR DESCRIPTION
**Pull request by HappyJuliana**
Снимок экрана 2023-08-29 181536

Снимок экрана 2023-08-29 181832

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,179 +1,228 @@
...
+            }
+
+            .sub-heading {
+                font-size: 1em;
+                line-height: 20px;
+            }
+
+            .button {
+                line-height: 15px;
+            }
```
Comment (emma13z) [inline index.html:190]: I would also adjust the padding and font-size of the buttons to make them look narrower on smaller screens. Maybe would adjust the headings' size as well. Not sure if it is a necessary thing though,  more likely my personal "design" preference I suppose :)

Comment (emma13z) [reply index.html:190]: The rest looks good to me!

**Review by emma13z — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (HappyJuliana) [reply index.html:190]: Headings are adjusted with the help of font-size and line-height --->
.main-heading {
                font-size: 2em;
                line-height: 40px;
            }
            .sub-heading {
                font-size: 1em;
                line-height: 20px;
            }
I've decided to do the same thing with the buttons, I set them "line-height: 15px;", but I forgot about font-size. I will definitely change it!
Changing font-size and line-height helped me to make headings and buttons look narrower on smaller screens.

Thank you for your review and "designal" commens <3

**Review by IvanCeban — APPROVED** [review]
Great job. Good namings. 
In future make the project files more structured. Put images in separate folder. For icons I highly recommend you use svg instead of png

**Pull request merged**